releases/goerli/1.9.6/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
+	<head>
+		<title>Goerli Release 1.9.6</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Goerli Release 1.9.6</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/goerli/1.9.6/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmbVo1yhEXpyq1ns6E4dc19sc4J5E9pdS8xHyTcoUQw1QW">Contract addresses (IPFS hosted)</a></li>
```

Comment:
The IPFS link should be updated with the latest version.

Reply:
done